test/rules/jsx-curly-spacing/never/test.tsx.lint

```diff
@@ -0,0 +1,65 @@
+const passA = <App foo={bar} />;
+
+const failA = <App foo={ bar } />;
+                       ~ [There should be no space after {]
+                             ~ [There should be no space before }]
+
+const failA2 = <App foo={bar } />;
+                             ~ [There should be no space before }]
+
+const failA3 = <App foo={ bar} />;
+                        ~ [There should be no space after {]
+
+const passB = <App foo={bar} {...bar} />;
+
+const failB = <App foo={ bar }  { ...bar } />;
+                       ~ [There should be no space after {]
+                             ~ [There should be no space before }]
+                                ~ [There should be no space after {]
+                                         ~ [There should be no space before }]
+
+const failB2 = <App foo={bar } {...bar } />;
+                             ~ [There should be no space before }]
+                                       ~ [There should be no space before }]
+
+const failB3 = <App foo={ bar} { ...bar} />;
+                        ~ [There should be no space after {]
+                               ~ [There should be no space after {]
+
+const failB4 = <App foo={bar} { ...bar } />;
+                              ~ [There should be no space after {]
+                                       ~ [There should be no space before }]
+
+const failB5 = <App foo={ bar } {...bar} />;
+                        ~ [There should be no space after {]
+                              ~ [There should be no space before }]
+
+const passC = <App foo={
+                        bar
+                        } {...bar} />;
+
+const passC2 = <App foo={
+                            bar
```

Comment:
This case was not covered by the rule. I corrected it so it is covered now. Thanks for pointing this out.